bagelbids/cli.py

```diff
@@ -36,6 +37,57 @@ def get_columns_about(data_dict: dict, concept: str) -> list:
     ]
 
 
+def map_categories_to_columns(data_dict: dict) -> dict:
+    """Create a mapping of categories to columns for a given data dictionary"""
+    return {
+        cat_name: get_columns_about(data_dict, cat_iri)
+        for cat_name, cat_iri in mappings.NEUROBAGEL.items()
+        if get_columns_about(data_dict, cat_iri)
```

Comment:
Agreed on being more explicit what this function does. 

> Doesn't this return statement assume that all the Neurobagel variables are categorical?

No. `mappings.Neurobagel` contains a list of categories we know about. Those are the same categories the annotation is (will be) asking about (e.g. `"Sex"`, `"Age"`, ...). When a user tells us during the annotation process that a given column (e.g. `column1`) contains information about one of these categories (e.g. `"Sex"`), then this is reflected in the data dictionary under the `IsAbout` attribute by setting the `TermURL` to the controlled term for this category (i.e. `bg:Sex`). This function just makes it easier to ask the inverse question of: "what columns (if any) have been assigned to the category `Sex`?".

So you go from
```json
{
  "column1": {
    "Annotations": {
      "IsAbout": {
        "TermURL": "bg:Sex",
        "Label": "Biological sex"
      }
    }
  }
}
```
to
```python
{"sex": ["column1"]}
```

That works whether the column is categorical or continuous.

Reply:
Ah okay, I think the function/variable naming confused me here - we are calling both continuous and categorical variables “categories”

Reply:
Yeah, I see the confusion. Categories are things like "Sex" or "Age" or "Diagnosis". A more appropriate word for them might be [Common Data Elements](https://www.nlm.nih.gov/oet/ed/cde/tutorial/03-100.html) - maybe we should adopt that. There is a one-to-many relationship between categories/CDEs and "columns" of a .tsv.

This obviously gets confusing because the columns can contain continuous or categorical values themselves. Maybe a better word for "categorical" in the context of columns would be "discrete"?